app/uk/gov/hmrc/fileupload/write/envelope/Envelope.scala

```diff
@@ -26,35 +26,11 @@ object EnvelopeHandler extends Handler[EnvelopeCommand, Envelope] {
 
   type CanResult = Xor[EnvelopeCommandNotAccepted, Unit.type]
 
-  def constraintsWithDefaultsIfNotProvided(constraintsO: Option[EnvelopeConstraintsO], defaultConstraints: EnvelopeConstraints): EnvelopeConstraints = {
-    val defaultMaxCapacity = defaultConstraints.maxNumFiles
-    val defaultMaxSize = defaultConstraints.maxSize
-    val defaultMaxSizePerItem = defaultConstraints.maxSizePerItem
-
-    constraintsO match {
-      case Some(consts) => {
-        EnvelopeConstraints(
-          maxNumFiles = consts.maxNumFiles.getOrElse(defaultMaxCapacity),
-          maxSize = consts.maxSize match {
-            case Some(maxSizeConst) => maxSizeConst
-            case _ => defaultMaxSize
-          },
-          maxSizePerItem = consts.maxSizePerItem match {
-            case Some(maxSizePerItemConst) => maxSizePerItemConst
-            case _ => defaultMaxSizePerItem
-          }
-        )
-      }
-      case _ => defaultConstraints
-    }
-
-  }
-
   override def handle = {
     case (command: CreateEnvelope, envelope: Envelope) =>
-      val constraints = constraintsWithDefaultsIfNotProvided(command.constraints, envelope.constraints)
-      envelope.canCreateWithFilesCapacityAndSize(constraints, envelope.constraints).map(_ =>
-        EnvelopeCreated(command.id, command.callbackUrl, command.expiryDate, command.metadata, constraints)
+      // val constraints = constraintsWithDefaultsIfNotProvided(command.constraints, envelope.constraints)
```

Comment:
this is the method i were taking about, i think @KorhanOzturk90 idea may be is to let me re-do the method?